### PR TITLE
Css position edit

### LINKED ISSUE #1443: [Edit] CSS Position: `position`
### Type of Edit (select all that apply)

Fix grammar, spelling, formatting, Clarify unclear content and/or terms, Add new content (definitions, codeblocks, etc.)

### Description (optional)

We'd like to make the following edits to the term entry about the CSS `position` property:

- Rewrite opening sentence to (1) explicitly name the term and (2) emphasize the term in **bold**.
- Currently, there are 4 "Example .." sections that showcase the different values of `position`. Can we consolidate this into 1 or 2 "Example .." sections that feature multiple `.box`es that have the same height/width but different `position` values? (`fixed`, `static`, `relative`, etc.)
- Can we add an image of what the rendered output of these examples would look like?
  - Make sure to save this image as a `.png` file and store in the [`media/` folder](https://github.com/Codecademy/docs/tree/main/media).

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by lindsaylearns**
### Description

Closes #1443 . Edited opening statement to explicitly state the property and bold the term. Consolidated the four examples into two examples, showing multiple ways to position a box. Added an image to show the output.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Dusch4593** [comment]
Thanks for submitting this, @lindsaylearns! 😄

I'll take a look at this for review 1️⃣

File: content/css/concepts/position/terms/position/position.md
Comment (Dusch4593) [inline content/css/concepts/position/terms/position/position.md:16]:
```suggestion
The **position** property defines how elements are placed within the document.
```

I think "placed" makes more sense

File: content/css/concepts/position/terms/position/position.md
Comment (Dusch4593) [inline content/css/concepts/position/terms/position/position.md:19]: Below, on line # 20, let's change from \```css to \```pseudo.

File: content/css/concepts/position/terms/position/position.md
Comment (Dusch4593) [reply content/css/concepts/position/terms/position/position.md:16]: Can we also edit the `Description` on line # 3 to say the following?

> Defines how elements are placed within the document.

File: content/css/concepts/position/terms/position/position.md
Comment (Dusch4593) [inline content/css/concepts/position/terms/position/position.md:64]:
```suggestion
Here's what the above examples look like:

![CSS Position Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-position-example.png)

The first box is sticky to the top of the document. The second is set to a fixed position 40px from the top of the page. The third box position is 80px from the element's top edge.
```

File: content/css/concepts/position/terms/position/position.md
Comment (Dusch4593) [inline content/css/concepts/position/terms/position/position.md:35]:
```suggestion
The following example features three `.box` elements with different `position` values:
```

I would place the latter sentences after the image you provided.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@lindsaylearns Left you some comments/suggestions 😄

**Comment by lindsaylearns** [comment]
@Dusch4593 I have made all requested changes. Thank you for your review!

**Comment by lindsaylearns** [comment]
@Dusch4593 Should I not rebase with the main branch when GitHub prompts? I'm happy to pull back and resubmit just my changes to this one file if that's preferred.

**Comment by Dusch4593** [comment]
> @Dusch4593 Should I not rebase with the main branch when GitHub prompts? I'm happy to pull back and resubmit just my changes to this one file if that's preferred.

I would advise against rebasing since that would overwrite parts of the commit history. Is this question in regards to keeping your Docs fork synced with the base `Codecademy/docs` repo?

**Comment by lindsaylearns** [comment]
> > @Dusch4593 Should I not rebase with the main branch when GitHub prompts? I'm happy to pull back and resubmit just my changes to this one file if that's preferred.
> 
> I would advise against rebasing since that would overwrite parts of the commit history. Is this question in regards to keeping your Docs fork synced with the base `Codecademy/docs` repo (for the `main` branch)?

Yes - when the message says the branch is out-of-date with the base branch, I can either update my branch or update with rebase. I definitely worded question wrong. I am asking if I should update my branch or just leave it alone.

**Comment by Dusch4593** [comment]
@lindsaylearns Ok gotcha. You are certainly welcome to click "Update branch" if you'd like (sometimes myself or another maintainer may do it, as well). Again, I would avoid rebasing by keeping this button set to "Update with merge commit":

Screen Shot 2023-01-06 at 6 23 35 PM

Ultimately, this branch will need to be in sync with the base `main` branch in order to be merged.

**Review by Dusch4593 — APPROVED** [review]
@lindsaylearns Thank you for implementing those changes! 😄

This looks good for the next review 👍🏻

**Review by SSwiniarski — APPROVED** [review]
@lindsaylearns, @Dusch4593 Looks good to me, I don't have anything to add.

**Comment by SSwiniarski** [comment]
@lindsaylearns, I did a npm format on the file to get the checks to pass.

@Dusch4593, anything else we need before we merge?

**Comment by Dusch4593** [comment]
Thank you for contributing to Codecademy Docs, @lindsaylearns! 🚀 🎊 🎉

The changes should now be live: https://www.codecademy.com/resources/docs/css/position/position

Currently, your profile seems to be anonymously listed on the entry page. To make it visible, you'll need to add your GitHub username to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account:

Screen Shot 2023-01-10 at 12 49 50 PM

Looking forward to your next Docs contribution! 😄

**Pull request merged**
